src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1

```diff
@@ -300,76 +288,77 @@ function Test-ValidateLogicApp
-
-	$locationName = 'northeurope'
+	$location = Get-Location "Microsoft.Logic" "workflows" "West US"
 
 	$definitionFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowDefinition.json"
 	$parameterFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowParameter.json"
 
-	$workflow = New-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $WORKFLOW_LOCATION -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
+	$workflow = New-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $location -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
 
 	# Test 1: Using physical file.
-	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $locationName -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
+	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $location -DefinitionFilePath $definitionFilePath -ParameterFilePath $parameterFilePath
 	
 	# Test 2: Using definition object and parameter file.
 	$definition = [IO.File]::ReadAllText($definitionFilePath)
-	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $locationName -Definition $definition -ParameterFilePath $parameterFilePath
-
-	# Test 3: Failure for an invalid definition.
-	try
-	{
-		Test-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Location $locationName -Definition '{}'
-	}
-	catch
-	{
-		Assert-AreEqual $_.Exception.Message "The request content is not valid and could not be deserialized: 'Required property '`$schema' not found in JSON. Path 'properties.definition', line 4, position 20.'."
-	}
+	Test-AzLogicApp -ResourceGroupName $resourceGroup.ResourceGroupName -Name $workflowName -Location $location -Definition $definition -ParameterFilePath $parameterFilePath
 
-	Remove-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Force
+	Remove-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -Force	
 }
 
 <#
 .SYNOPSIS
-Test New-AzLogicApp to create logic app for non-existing service plan. Constraint validation.
+Test Get-AzLogicAppUpgradedDefinition to generate an upgraded definition for a workflow of older schema.
 #>
-function Test-CreateLogicAppWithNonExistingAppServicePlan
+function Test-GetUpgradedDefinitionForLogicApp
 {
 	$resourceGroup = TestSetup-CreateResourceGroup
 	$workflowName = getAssetname
 	$resourceGroupName = $resourceGroup.ResourceGroupName
+	$location = Get-Location "Microsoft.Logic" "workflows" "West US"
+
 	$definitionFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowDefinition.json"
 	$parameterFilePath = Join-Path $TestOutputRoot "Resources" "TestSimpleWorkflowParameter.json"
-	$Plan = "B9F87338CAE4470F9116F3D685365748"
-	try
```

Comment:
Why is this removed?

Reply:
This is more of the dud App Service plan code. This try catch doesn't even actually use the App Service plan in the New-AzLogicApp, so the catch would never fire